### DISCUSSION TITLE
Constellation lines

### DISCUSSION THREAD

**Discussion by AdrianTolley** [Stellarium/stellarium#3677] [General]
Hello - I just downloaded the latest version of Stellarium, because I realised the version I'd been using was out of date.  I have just tried to set up the constellation lines, it helps me find 'where' an object is in the night sky.  I can't seem to make it work.  I go into the 'Sky Culture' tab, tick the box that says 'Constellation lines' and then they appear.  BUT - they do not 'save' when I close down Stellarium and open it again, and I have to repeat the whole process.  Is this a bug, or is it me doing something stupid?  There seems no way of 'saving' the option.  I managed to set up other things OK, like location and scenery.  I am not a very 'tec' person, I just like using Stellarium when I see something in night sky from my home and wonder what it is.  Many thanks. - Adrian.

**Comment by alex-w** [comment]
Did you press button "Save setting" in Configuration dialog [F2] / Main after changes?

**Reply by AdrianTolley** [reply]
Ah that fixed it!  Thank you.

**Comment by gzotti** [comment]
If you regularly use the program, you may also familiarize yourself with a few time saving keyboard shortcuts. E.g., press "C" for Constellations. The keys are shown when going over the respective button on the lower bar.

**Reply by AdrianTolley** [reply]
Great, thanks very much
